### DISCUSSION TITLE
PRG2BIN Error

### DISCUSSION THREAD

**Discussion by ccantrell72** [fdbozzo/foxbin2prg#101] [Q&A]
Error 36, Command contains unrecognized phrase/keyword.
setparsedinfoline, 21214
DO c:\develop\3rdparty\thor\thor\tools\components\foxbin2prg\foxbin2prg.exe WITH "PRG2BIN", "C:\Develop\Project.pj2"

**Comment by lscheffler** [comment]
@ccantrell72 
Do you see what file of the pj2 caused this? And possibly post it?

**Reply by lscheffler** [reply]
@ccantrell72   
What version of FoxBin2Prg are you using?  
- Run `DO FOXBIN2PRG.EXE WITH "VERNO"`
- open FoxBin2Prg.prg and look up the value of *c_FB2PRG_Version_Real*

I checked the code right now.  There is a macro expansion gone wrong.   
I guess I need the text file raising the error. Please proceed like:
1. Set a local `Try Catch` around  the line.
2. Post the values of  `tcInfoLine`, `lcAsignacion`
3. Identify the text file in the call stack
4. Post the text file, if possible

**Reply by ccantrell72** [reply]
It ended up being a value in this:  	*<.User = '' />

So we blanked it and it fixed.

**Reply by lscheffler** [reply]
@ccantrell72   
Hi, greatings from SWFox 2023   
Thank you for the information. 

If you recreate the text file from the binary, does the error recreates itself? Or how was the error in the file created the first place?

**Reply by ccantrell72** [reply]
Creating from SWFox 2023 too

**Reply by lscheffler** [reply]
(\*.\*)

**Comment by lscheffler** [comment]
@ccantrell72 Can we close this?
